core/templates/pages/admin-page/roles-tab/role-graph.directive.html

```diff
@@ -173,9 +173,9 @@
         <form class="form-horizontal" ng-submit="$ctrl.submitAddCommunityReviewerForm($ctrl.formData.addCommunityReviewer)">
           <legend class="text-center">Add Reviewer For Community</legend>
           <div class="d-xl-inline-flex form-group">
-            <label class="col-md-4 col-lg-4 col-sm-4 col-form-label">Enter Username</label>
+            <label for="add-community-reviewer-input" class="col-md-4 col-lg-4 col-sm-4 col-form-label">Enter Username</label>
```

Comment:
username-input (Will it be fine to have two same for value in one page?)

Reply:
They need to be slightly different but I'll change it a little